scripts/test/procedures.py

```diff
@@ -1,13 +1,29 @@
 import shutil
 import subprocess
 from pathlib import Path
+import sys
 
-from scripts._job import Step, run_job
+
+def create_mypy_pkg_file():
+    pkg_path = [x for x in sys.path if x.endswith('site-packages')]
+
+    if not Path(fr'{pkg_path[0]}/my_path.pth').exists():
+        with open(fr'{pkg_path[0]}/my_path.pth', 'w') as file:
+            file.write(str(Path.cwd()))
+
+
+def destroy_mypy_pkg_file():
+    pkg_path = [x for x in sys.path if x.endswith('site-packages')]
+
+    if Path(fr'{pkg_path[0]}/my_path.pth').exists():
+        Path(fr'{pkg_path[0]}/my_path.pth').unlink()
 
 
 def run_mypy_src():
+    create_mypy_pkg_file()
     cmd = ["mypy", "pandas-stubs", "tests", "--no-incremental"]
     subprocess.run(cmd, check=True)
```

Comment:
If for some reason this fails, the file would still be there.  Can't you use the `rollback` technique you have used elsewhere in case of failure?